modules/gitops-argocd-cli-creating-an-application-in-core-mode.adoc

```diff
@@ -0,0 +1,142 @@
+// Module included in the following assemblies:
+//
+// * argocd_applications/creating-an-application-using-gitops-argocd-cli.adoc
+// * declarative_clusterconfig/configuring-an-openshift-cluster-by-deploying-an-application-with-cluster-configurations.adoc
+
+ifeval::["{context}" == "configuring-an-openshift-cluster-by-deploying-an-application-with-cluster-configurations"]
+:cluster:
+endif::[]
+ifeval::["{context}" == "creating-an-application-using-gitops-argocd-cli"]
+:app:
+endif::[]
+
+:_mod-docs-content-type: PROCEDURE
+[id="gitops-argocd-cli-creating-an-application-in-core-mode_{context}"]
+= Creating an application in core mode by using the GitOps CLI
+
+You can create applications in `core` mode by using the {gitops-shortname} `argocd` CLI.
+
+ifdef::cluster[]
+This sample workflow walks you through the process of configuring Argo CD to recursively sync the content of the `cluster` directory to the `cluster-configs` application. The directory defines the {OCP} cluster configurations and the `spring-petclinic` namespace on the cluster.
+endif::cluster[]
+
+.Prerequisites
+
+* You have installed the {gitops-title} Operator in your cluster.
+* You have installed the OpenShift CLI (`oc`).
+* You have installed the {gitops-title} `argocd` CLI. 
+
+.Procedure
+
+. Log in to the {OCP} cluster using the `oc` CLI tool:
++
+[source,terminal]
+----
+$ oc login -u <username> -p <password> <server_url>
+----
++
+.Example
+[source,terminal]
+----
+$ oc login -u kubeadmin -p '<password>' https://api.crc.testing:6443
+----
+
+. Check whether the context is set correctly in the `kubeconfig` file:
++
+[source,terminal]
+----
+$ oc config current-context
+----
+
+. Set the default namespace of the current context to `openshift-gitops`:
++
+[source,terminal]
+----
+$ oc config set-context --current --namespace openshift-gitops
+----
+
+. Set the following environment variables to override the Argo CD component names:
++
+[source,terminal]
+----
+$ export ARGOCD_REPO_SERVER_NAME=openshift-gitops-repo-server
+----
+
+. Verify that you are able to run `argocd` commands in `core` mode by listing all applications:
++
+[source,terminal]
+----
+$ argocd app list --core
+----
++
+If the configuration is correct, then existing applications will be listed with the following header:
++
+.Sample output
+[source,terminal]
+----
+NAME CLUSTER NAMESPACE  PROJECT  STATUS  HEALTH   SYNCPOLICY  CONDITIONS  REPO PATH TARGET
+----
+
+ifdef::cluster[]
+. Create an application in `core` mode:
++
+[source,terminal]
+----
+$ argocd app create app-cluster-configs --core \
+    --repo https://github.com/redhat-developer/openshift-gitops-getting-started.git \
+    --path cluster \
+    --revision main \
+    --dest-server  https://kubernetes.default.svc \
+    --dest-namespace spring-petclinic \
+    --directory-recurse checked\
```

Comment:
remove the word `checked`. It should just be `--directory-recurse`. Checked is applicable only for Web Console where there is a check box and user has to check it.